### ISSUE TITLE
"npm view ... versions" broken with version bounds.

### ISSUE THREAD

**Issue by damncabbage** [npm/npm#12538]
With:
* npm v3.8.8 (just did an `npm install -g npm` to upgrade), and
* Node v4.1.0 and v6.0.0

... Adding a version bound to a `npm view ... versions` results in strange output:

```
$ npm view purescript@'>0.5.0' versions
[
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {},
  {}
]
```

This works without the version bound, eg.
```
$ npm view purescript versions
[ '0.0.0',
  '0.0.1',
  '0.1.0',
  '0.6.9',
  '0.6.10',
  '0.6.11',
  '0.7.1',
  '0.7.2',
  '0.7.3-fixed',
  '0.7.4-rc.1',
  '0.7.4-rc.2',
  '0.7.4',
  '0.7.5-rc.1',
  '0.7.5-rc.2',
  '0.7.5-rc.3',
  '0.7.5-rc.4',
  '0.7.5',
  '0.7.6-rc.1',
  '0.7.6',
  '0.8.0-rc.1',
  '0.8.0-rc.2',
  '0.8.0',
  '0.8.1-rc.1',
  '0.8.2-rc.1',
  '0.8.2',
  '0.8.3-rc.1',
  '0.8.3',
  '0.8.4-rc.1',
  '0.8.4',
  '0.8.5-rc.1',
  '0.8.5',
  '0.8.6-rc.1' ]
```

**Comment by watilde** [comment]
Hi Rob,
Thanks for reporting an issue. This issue will be fixed by next version **v3.8.9**. Patch is here: https://github.com/npm/npm/pull/12450

**Comment by damncabbage** [comment]
Ah! Thanks. My apologies, I couldn't find any issue mentioning this when I went looking. :sweat_drops: 

(Closing this now.)
